recipes/spirv-tools/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SpirvtoolsConan(ConanFile):
+    name = "spirv-tools"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Tools/"
+    description = "SPIRV-Tools "
+    topics = ("conan", "spirv", "spirv-v", "vulkan", "opengl", "opencl", "hlsl", "khronos")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    exports_sources = ["CMakeLists.txt"]
+    license = "Apache-2.0"
+    generators = "cmake"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "c_only": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "c_only": False
+    }
+
+    def requirements(self):
+        self.requires.add("spirv-headers/1.5.1")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "SPIRV-Tools-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+
+        # Required by the project's CMakeLists.txt
+        cmake.definitions["SPIRV-Headers_SOURCE_DIR"] = self.deps_cpp_info["spirv-headers"].rootpath
+
+        # There are some switch( ) statements that are causing errors
+        # need to turn this off
+        cmake.definitions["SPIRV_WERROR"] = False
+
+        cmake.configure(build_folder=self._build_subfolder)
+        return cmake
+
+    def build(self):
```

Comment:
In CMakeLists, SPIRV-tools has `set(CMAKE_POSITION_INDEPENDENT_CODE ON)`. It should be removed.

Reply:
So would I have to make a patch that would delete that line from the CMakeLists.txt file?